src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { Avatar, Box, Flex, Text } from "@chakra-ui/react";
+
+export const Sidebar = () => {
+  const gamesPattern = [
+    {
+      teamName: "RNG",
+      teamLogo: "",
+      gameDate: "Sexta feira",
+      championship: "Mundial Internacional",
+      gameHour: "20:00",
+    },
+    {
+      teamName: "JDG",
+      teamLogo: "",
+      gameDate: "Sexta feira",
+      championship: "Mundial Internacional",
+      gameHour: "20:00",
+    },
+    {
+      teamName: "Pain",
+      teamLogo: "",
+      gameDate: "Sexta feira",
+      championship: "Mundial Internacional",
+      gameHour: "20:00",
+    },
+    {
+      teamName: "INTZ",
+      teamLogo: "",
+      gameDate: "Sexta feira",
+      championship: "Mundial Internacional",
+      gameHour: "20:00",
+    },
+    {
+      teamName: "INTZ",
+      teamLogo: "",
+      gameDate: "Sexta feira",
+      championship: "Mundial Internacional",
+      gameHour: "20:00",
+    },
+    {
+      teamName: "INTZ",
+      teamLogo: "",
+      gameDate: "Sexta feira",
+      championship: "Mundial Internacional",
+      gameHour: "20:00",
+    },
+    {
+      teamName: "INTZ",
+      teamLogo: "",
+      gameDate: "Sexta feira",
+      championship: "Mundial Internacional",
+      gameHour: "20:00",
+    },
+    {
+      teamName: "INTZ",
+      teamLogo: "",
+      gameDate: "Sexta feira",
+      championship: "Mundial Internacional",
+      gameHour: "20:00",
+    },
+  ];
+  return (
+    <Flex
+      bg="brand"
+      w={["100%", "100%", "25%", "25%"]}
+      h="100vh"
+      flexDirection="column"
+      justifyContent="space-around"
+      overflow="auto"
+    >
+      {gamesPattern.map((gamePattern) => (
+        <Flex
+          key={null}
+          flexDirection="column"
+          alignItems="center"
+          bg="#D9D9D9"
```

Comment:
Essas cores me incomoda, pq são cores do figma e você tá colocando hardcoded. Quando for mudar vai ter que mudar em vários lugares em vez de mudar em um só: no tema

Reply:
vou add ao theme👍🏻